fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -34,7 +32,7 @@
  * @author Gregory Jansen
  */
 public class FedoraUserSecurityContext implements SecurityContext,
-        AdvancedAuthorizationProvider {
+AdvancedAuthorizationProvider {
```

Comment:
Generally, reformatting that has nothing to do with functional updates is frowned upon for the reason that it distracts the code review process, as well as makes code-history archaeology more difficult.